### DISCUSSION TITLE
Animax India

### DISCUSSION THREAD

**Discussion by GiovanniSebastian** [iptv-org/community#931] [Q&A]
### Channel ID

Animax.in

### Stream URL (optional)

https://www.beastplayer.tk/hls/arturojio/player1.php?id=2258&slug=Animax

### Notes (optional)

Not the Asian feed. It's a different feed from India.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by R4IN2007** [comment]
https://www.beastplayer.tk/hls/arturojio/app/live36af421521ec562e56b8f671c573143718038713fab1b684960bd0aa3082a4f9facdc6929c69f0a762127636c6d885e0b7e81dfd5249619fbdb3840123700b731bdcbd879c8ffb59f07897fc7080c80b/2258/Animax/index.m3u8

**Comment by snh24github** [comment]
added the star plus channel for india

**Comment by freearhey** [comment]
https://www.beastplayer.tk/hls/arturojio/app/live36af421521ec562e56b8f671c573143718038713fab1b684960bd0aa3082a4f9facdc6929c69f0a762127636c6d885e0b7e81dfd5249619fbdb3840123700b731bdcbd879c8ffb59f07897fc7080c80b/2258/Animax/index.m3u8 (offline)
<img width="845" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/12704f5b-32ef-46c9-9fd5-5e1f2aaa3800">
